addons/website/views/snippets/s_contact_info.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_contact_info" name="Contact Info">
+    <section class="s_contact_info pt48 pb64 o_cc o_cc1">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="11">
+                <div class="o_grid_item g-col-lg-6 g-height-5 col-lg-6 col-10 offset-1 order-lg-0" style="order: 2;--grid-item-padding-x: 0px; --grid-item-padding-y: 24px; grid-area: 1 / 1 / 6 / 7; z-index: 1;" data-name="Title block">
+                    <h2>Contact Us</h2>
+                    <p class="lead">We'd love to hear from you! If you have any questions, feedback, or need assistance, please feel free to reach out to us using the contact details provided. Our team is here to help and will respond as soon as possible. Thank you for getting in touch!</p>
+                </div>
+                <div class="o_grid_item g-col-lg-6 g-height-2 col-lg-6 col-10 offset-1 order-lg-0" style="order: 3; --grid-item-padding-x: 0px; --grid-item-padding-y: 16px; grid-area: 6 / 1 / 8 / 7; z-index: 2;" data-name="Info block">
+                    <h3 class="h5-fs">
+                        <i class="fa fa-fw fa-envelope-o" role="presentation"/>
+                        Email
+                    </h3>
+                    <p>
+                        &#160;&#160;&#160;&#160;&#160;&#160;&#160;<a href="mail-to:info@yourcompany.example.com" title="Send an email">info@yourcompany.example.com</a>
```

Comment:
1) The link won't work, the correct link format is "mailto:" without the dash.
2) These hardcoded HTML entities hurt... but maybe we don't have other possibilities in this case? @stefanorigano

Reply:
Hello @SergeBayet ,
the hardcoded HTML entities method it's valid 'cause it's the more predictable and easy for the user to edit.

Still, in this specific case, it leads to a suboptimal visual result on mobile, I agree..  I downplayed the scenario during my review, thanks for pointing it out 👍 

@xlu-odoo , I propose a simpler approach...

```xml
<icon/> <br/>
<link/>
```
... if the user wants to achieve what we initially had in mind he can still do that (probably reviewing the font size to handle mobile correctly ).

Thanks!